static/app/components/u2f/u2finterface.tsx

```diff
@@ -57,7 +62,17 @@ class U2fInterface extends React.Component<Props, State> {
             hasBeenTapped: true,
           },
           () => {
-            const u2fResponse = JSON.stringify(data);
+            let u2fResponse = JSON.stringify(data);
+            // below is to check if its a webauthn returned object which is a AuthenticatorAssertionResponse
+            if (typeof data.response !== 'undefined') {
```

Comment:
do you have to use typeof here? can you just do an implicit truthy check? `if (data.response)`?